API/Configuration/MappingProfile.cs

```diff
@@ -62,14 +63,31 @@ public MappingProfile()
                 .ForMember(q => q.Name, opt => opt.MapFrom(q => q.FollowedUser.Name))
                 .ForAllOtherMembers(o => o.Ignore());
 
-           CreateMap<UserProject, UserProjectResourceResult>()
+            CreateMap<UserProject, UserProjectResourceResult>()
                 .ForMember(q => q.Id, opt => opt.MapFrom(p => p.Project.Id))
                 .ForMember(q => q.Name, opt => opt.MapFrom(p => p.Project.Name))
                 .ForMember(q => q.ShortDescription, opt => opt.MapFrom(p => p.Project.ShortDescription))
                 .ForMember(q => q.Uri, opt => opt.MapFrom(p => p.Project.Uri))
                 .ForMember(q => q.Description, opt => opt.MapFrom(p => p.Project.Description))
                 .ForAllOtherMembers(o => o.Ignore());
 
+            CreateMap<Portfolio, PortfolioResourceResult>()
+               .ForMember(q => q.Id, opt => opt.MapFrom(p => p.Id))
+               .ForMember(q => q.Template, opt => opt.MapFrom(p => p.Template))
+               .ForMember(q => q.Title, opt => opt.MapFrom(p => p.Title))
+               .ForMember(q => q.UserId, opt => opt.MapFrom(p => p.User.Id))
+               .ForMember(q => q.UserName, opt => opt.MapFrom(p => p.User.Name))
+               .ForMember(q => q.PortfolioItem, opt => opt.MapFrom(p => p.PortfolioItem))
```

Comment:
This will fail since Portfolio.Portfolio item is a different list type then that of PortfolioResourceResult. You will notice this once the projectitems are included, they are empty as of now.